src/OrchardCore/OrchardCore.DisplayManagement/Implementation/DefaultHtmlDisplay.cs

```diff
@@ -50,6 +50,16 @@ public async Task<IHtmlContent> ExecuteAsync(DisplayContext context)
             // Check if the shape is pre-rendered.
             if (shape is IHtmlContent htmlContent)
             {
+                if (htmlContent is PositionWrapper wrapper)
```

Comment:
Example of things to try with this change: Widgets, FlowPart, BagPart, GraphQL rendering, layouts, custom placement.

Reply:
It would not break anything in OC, rather fixes Blazor component rendering. 

This PR  returns the original pre-rendered IHtmlContent when the shape provided in DisplayAsync is PositionWrapper.

PositionWrapper is nothing but holds pre-rendered IHtmlContent, and it can only contain one IHtmlContent.  Hence there no need of nesting PositionWrapper in PositionWrapper. 

Currently In ThemeLayout we use PositionWrapper to pre-render (while maintaining position on zone) before we place on Layout. This allows to have lazy placed zones to be rendered on Layout but requires us to call DisplayAsync twice, that unnecessary nests PositionWrapper in PositionWrapper.